src/rules/gl.toml

```diff
@@ -0,0 +1,40 @@
+min_trimmed_length = 3
+min_word_count = 2
+max_word_count = 14
+min_characters = 1
+may_end_with_colon = false
+quote_start_with_letter = true
+needs_punctuation_end = false
+needs_letter_start = true
+needs_uppercase_start = true
+disallowed_symbols = [
```

Comment:
You might want to also have a look at `allowed_symbols_regex`, might make this easier.